site/source/components/PlaceDesEntreprises.tsx

```diff
@@ -0,0 +1,114 @@
+import { lazy, Suspense, useEffect } from 'react'
+import styled from 'styled-components'
+import Emoji from './utils/Emoji'
+import { iframeResize } from 'iframe-resizer'
+import { PopoverWithTrigger } from '@/design-system'
+import { Button } from '@/design-system/buttons'
+import { Loader } from '@/design-system/icons/Loader'
+import { Body } from '@/design-system/typography/paragraphs'
+import { Trans, useTranslation } from 'react-i18next'
+import { isProduction } from '@/utils'
+
+const Iframe = styled.iframe`
+	width: 1px;
+	min-width: 100%;
+	height: 80vh;
+`
+
+const IframeContainer = styled.div`
+	margin: 0 -3rem;
+`
+
+export const PlacesDesEntreprisesIframe = ({ src }: { src: string }) => {
+	useEffect(() => {
+		iframeResize({}, '#pdeIframe')
+	}, [])
+
+	return (
+		<IframeContainer>
+			<Iframe
```

Comment:
Pas bloquant pour merger mais on a le même problème que sur notre intégration iframe : quand on clique sur un lien à l'intérieur de l'iframe, la position de scroll ne remonte pas en haut de la “page” #1348

Reply:
J'ai essayé un fix pour scroll to top sur l'onLoad de l'iframe, j'attend le build pour vérifier que ça fonctionne bien

Reply:
ça fonctionne bien !